src/main/java/core/basesyntax/dao/WarehouseImpl.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.dao;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class WarehouseImpl implements Warehouse {
+    private final Map<String, Integer> listItems = new HashMap<>();
```

Comment:
It's okay that you are using Map, But let's create model Fruit and use it as key to this map, even though it will have only one field